src/server/passport/strategies.js

```diff
@@ -30,6 +32,13 @@ const loginStrategy = (UserModel = require('../models/user'), strategyUtils = re
 					return validation
 				})
 		}
+
+		function isUserVerified(validation){
+			if(!validation.user.hasOwnProperty('active') || !validation.user.active)
```

Comment:
Is there a chance that the `validation.user` will inherit `active` in prototype chain?
Why is it wrong if it does?

Reply:
BTW you can get rid of negation
```
if(validation.user.hasOwnProperty('active') && validation.user.active) return validation
throw new Error(USER_NOT_VERIFIED)
```

Reply:
Those OR flags should be AND, that's my bad, I'll need to discuss it further with you about what you're asking regarding the active flag. I'm working under the assumption that validation.user is a User model, which will contain the "active" (verified) flag. As for the removal of negation, I'm pretty sure I need to return validation to continue the pipeline, since finish() requires it. 

```return Promise.resolve(validation)
			.then(assignFoundUser)
			.then(assignPasswordValidationResult)
			.then(isUserVerified)
			.then(finish)
			.catch(() => {
				const userToSerialize = null
				done(responseManager.MESSAGES.ERRORS.UNAUTHORIZED, userToSerialize)
			})
```